apps/tx-builder/src/components/Builder.tsx

```diff
@@ -123,6 +123,15 @@ export const Builder = ({
     setInputCache(inputCache.slice());
   };
 
+  const isInputValueValid = useCallback((val: string) => {
```

Comment:
You can define this function outside the component body and you won't need `useCallback`

Reply:
yes I can do it.

In this concrete component there are a lot of functions created inside and outside but there is no concrete pattern for doing it. What I'm doing in this issue #214  is extracting the functions from outside the component to the utils file. What do you think if I do the same with this?

Reply:
I think it's OK to keep functions in a single file if you don't use them in other files. No strong preference, though.

> there is no concrete pattern for doing it

well, besides the pattern, using it with useCallback is less efficient because the memoization isn't free

Reply:
yes, with "pattern" i mean in this concrete component there are more useCallback uses that doesn't make sense for the same memoization issues you are mentioning and should be extracted as well. I will do it in the other issue i'm developing.

I extracted this function to outside anyway. Thanks !!